isserviceup/notifiers/slack.py

```diff
@@ -0,0 +1,19 @@
+from decouple import config
+import requests
+from isserviceup.notifiers.notifier import Notifier
+
+SLACK_WEBHOOK_URL = config('SLACK_WEBHOOK_URL', default=None)
```

Comment:
let's keep all the config variables in the main config file, you want to pass the variable to the notifier constructor